docs/src/content/overview-features.md

```diff
@@ -51,22 +127,19 @@ The substituted URL is fetched instead of the original resource
 and the corresponding HTTP response is returned transparently to the client.
 Note that if the original destination uses HTTP2, the substituted destination
 needs to support HTTP2 as well, otherwise the substituted request may fail.
-`map_remote` patterns looks like this:
+`map_remote` patterns look like this:
 
 ```
-|flow-filter|regex|replacement
-|flow-filter|regex|@file-path
-|regex|replacement
-|regex|@file-path
+|flow-filter|url-regex|replacement
```

Comment:
What about putting the optional filter at the end? Not sure how the discussion in #3948 ended, it's the same in #4060

Reply:
There are three arguments in favor of keeping them at the start:

- It matches the structure of natural language, e.g. "Only for flows matching X replace Y with Z".
- For some reason we often use `/` as the separator in examples - maybe it's inspired by sed, I don't know. In any case, this clashes with replacement paths that contain slashes ( `/~m GET/foo/@~/foo.bin` works as expected, if we change argument order, `/foo/@~/foo.bin/~m GET` would be parsed wrongly)
- Existing patterns continue to work as-is.

None of these arguments is particularly strong, but neither are the arguments for moving them to the end. I don't have a very strong opinion here in general, if you all feel strongly we can move them.

Reply:
No feelings at all, just making sure it wasn't overlooked as you were originally [in favor of it](https://github.com/mitmproxy/mitmproxy/issues/3948#issuecomment-648932541)

Reply:
Yeah I brought this up but agree with Max that its better to leave it as it is.